### DISCUSSION TITLE
Add Stryker in Simple Icons

### DISCUSSION THREAD

**Discussion by regseb** [stryker-mutator/stryker-mutator.github.io#422] [Show and tell]
I created a **[pull request](https://github.com/simple-icons/simple-icons/pull/7388)** on [Simple icons](https://simpleicons.org/) to add Stryker logo. The icon will be used in a [Shields.io](https://shields.io/) badge with parameter: `?logo=stryker`.

**Comment by hugo-vrijswijk** [comment]
Very cool! Hope it gets merged soon!

**Comment by regseb** [comment]
The pull request will be refused because Stryker doesn't meet the popularity criteria: _the amount of GitHub "stars" should be above 5k._ 😞
Except to have a strong argument why it should nevertheless be considered popular enough. I didn't find any. If you have one, you can answer in the pull request: https://github.com/simple-icons/simple-icons/pull/7388
